dashboard/app/models/pd/application/application_base.rb

```diff
@@ -105,6 +106,10 @@ def set_type_and_year
       self.application_type = nil
     end
 
+    def update_accepted_date
+      self.accepted_at = status == 'accepted' ? Time.now : nil
+    end
```

Comment:
how about, above on line 100
```ruby
before_save :update_accepted_date, if: -> {status_changed? && accepted?}
```

Then this method can be simplified to:
```ruby
def update_accepted_data
  self.accepted_at = Time.zone.now
end
```

Reply:
I want accepted date to be cleared if the application is un-accepted